lib/core/presentation/pages/event/event_control_panel_page/sub_pages/event_location_setting_page/sub_pages/event_location_setting_detail_page.dart

```diff
@@ -69,207 +71,130 @@ class _EventLocationSettingDetailPageState
+                                    additionalDirections: value,
+                                  ),
+                                );
+                          },
                         ),
-                      ),
+                        SizedBox(height: Spacing.smMedium * 2),
+                        _buildConfirmLocationButton(),
+                      ],
                     ),
-                    _buildSaveButton(),
-                  ],
+                  ),
                 ),
-              );
-            },
-          ),
-        ),
-      ),
-    );
-  }
-
-  _onTapEnterAddress() async {
-    // show input autocomplete with selected mode
-    // then get the Prediction selected
-    final p = await PlacesAutocomplete.show(
-      context: context,
-      apiKey: AppConfig.googleMapKey,
-      onError: (response) {
-        ScaffoldMessenger.of(context).showSnackBar(
-          SnackBar(
-            content: Text(response.errorMessage ?? 'Unknown error'),
+              ),
+            ],
           ),
         );
       },
-      mode: Mode.overlay,
-      resultTextStyle: Theme.of(context).textTheme.titleMedium,
-    );
-    await displayPrediction(
-      p != null ? google_places_service.Prediction.fromJson(p.toJson()) : null,
-      ScaffoldMessenger.of(context),
     );
   }
 
-  Future<void> displayPrediction(
-    google_places_service.Prediction? p,
-    ScaffoldMessengerState messengerState,
-  ) async {
-    if (p == null) {
-      return;
-    }
-    FocusScope.of(context).requestFocus(FocusNode());
-    // Get place detail (lat/lng)
-    final places = google_places_service.GoogleMapsPlaces(
-      apiKey: AppConfig.googleMapKey,
-      apiHeaders: await const GoogleApiHeaders().getHeaders(),
-    );
-    final detail = await places.getDetailsByPlaceId(p.placeId!);
-    context
-        .read<EventLocationSettingBloc>()
-        .add(PlaceDetailsChanged(placeDetails: detail.result));
-  }
-
-  _buildSaveButton() {
+  _buildConfirmLocationButton() {
     return BlocBuilder<EventLocationSettingBloc, EventLocationSettingState>(
       builder: (context, state) {
-        return Padding(
-          padding: EdgeInsets.only(bottom: 15.h),
-          child: LinearGradientButton(
-            label: widget.address != null
-                ? t.common.actions.edit
-                : t.common.actions.add,
-            height: 48.h,
-            radius: BorderRadius.circular(24),
-            textStyle: Typo.medium.copyWith(),
-            mode: state.placeDetailsText == ''
-                ? GradientButtonMode.lavenderDisableMode
-                : GradientButtonMode.lavenderMode,
-            onTap: () {
-              Vibrate.feedback(FeedbackType.light);
-              context
-                  .read<EventLocationSettingBloc>()
-                  .add(const SubmitAddLocation());
-            },
-            loadingWhen: state.status.isInProgress,
-          ),
+        return LinearGradientButton(
```

Comment:
use LinearGradient.primary for consistency, 
Don't need to add custom height, mode, or radius